.github/workflows/DEPLOY.yaml

```
@@ -0,0 +1,156 @@
+            secret/data/products/connectors/ci/common ARTIFACTORY_PSW;
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+          cache: 'maven'
+          server-id: camunda-nexus
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+
+      - name: Build Connectors Artifacts
+        run: mvn -B compile generate-sources source:jar javadoc:jar deploy
+        working-directory: connector/
+        env:
+          MAVEN_USERNAME: ${{ steps.secrets.outputs.ARTIFACTORY_USR }}
+          MAVEN_PASSWORD: ${{ steps.secrets.outputs.ARTIFACTORY_PSW }}
+
+      - name: Build Bundle Artifact
+        run: mvn -B compile generate-sources source:jar javadoc:jar deploy
+        working-directory: docker/bundle/mvn/
+        env:
+          MAVEN_USERNAME: ${{ steps.secrets.outputs.ARTIFACTORY_USR }}
+          MAVEN_PASSWORD: ${{ steps.secrets.outputs.ARTIFACTORY_PSW }}
+
+  build-images:
+    name: Build image and push SNAPSHOT Docker image from main
+    runs-on: ubuntu-latest
+    needs: deploy-snapshots
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Lint Connector Runtime Dockerfile
+        uses: hadolint/hadolint-action@v2.0.0
+        with:
+          dockerfile: docker/runtime/Dockerfile
+
+      - name: Lint Connector Bundle Dockerfile
+        uses: hadolint/hadolint-action@v2.0.0
+        with:
+          dockerfile: docker/bundle/bash/Dockerfile
+
+      - name: Lint Connector Bundle Dockerfile
+        uses: hadolint/hadolint-action@v2.0.0
+        with:
+          dockerfile: bundle/mvn/camunda-saas-bundle/Dockerfile
+
+      - name: Lint Connector Bundle Dockerfile
+        uses: hadolint/hadolint-action@v2.0.0
+        with:
+          dockerfile: docker/bundle/mvn/default-bundle/Dockerfile
+
+      - name: Import Secrets
+        id: secrets
+        uses: hashicorp/vault-action@v2.4.2
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+          exportEnv: false # we rely on step outputs, no need for environment variables
+          secrets: |
+            secret/data/products/connectors/ci/common DOCKERHUB_USER;
+            secret/data/products/connectors/ci/common DOCKERHUB_PASSWORD;
+            secret/data/products/connectors/ci/common ARTIFACTORY_USR;
+            secret/data/products/connectors/ci/common ARTIFACTORY_PSW;
+
+      - name: Setup Java Build
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+          cache: 'maven'
+          server-id: camunda-nexus
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+
+      - name: Deploy Maven Artifact
+        run: mvn -B compile generate-sources source:jar javadoc:jar install # TODO : Switch back to deploy
```

Since we have the Connectors in here now as well, let's switch to deploy 👍 
```suggestion
        run: mvn -B compile generate-sources source:jar javadoc:jar deploy
```